src/AppProviders.tsx

```diff
@@ -56,32 +57,31 @@ export function AppProviders({ children }: ProvidersProps) {
         <PluginProvider>
           <ColorModeScript storageKey='ss-theme' />
           <ChatwootWidget />
+          <Web3Modal projectId={getConfig().REACT_APP_WALLET_CONNECT_PROJECT_ID} />
```

Comment:
Not strictly part of this fix, but was hyper confusing because it added a walletconnect ethereum provider that doesnt seem to be used at all.

Even if it is used, its likely broken for most chains because it was hardcoded to eth mainnet